lib/util/todo_tile.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_slidable/flutter_slidable.dart';
+
+class ToDoTile extends StatelessWidget {
+  final String taskname;
+  final bool taskStatus;
+  Function(bool?)? onChanged;
+  Function(BuildContext)? deleteFunction;
+  ToDoTile({
+    super.key,
+    required this.taskname,
+    required this.taskStatus,
+    required this.onChanged,
+    required this.deleteFunction,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 20.0, top: 20.0, right: 20.0),
+      child: Slidable(
+        endActionPane: ActionPane(
+          motion: const StretchMotion(),
+          children: [
+            SlidableAction(
+              onPressed: deleteFunction,
+              backgroundColor: Colors.red.shade700,
+              icon: Icons.delete,
+              borderRadius: BorderRadius.circular(12.0),
+            )
+          ],
+        ),
+        child: Container(
+          padding: const EdgeInsets.all(14.0),
+          child: Row(
+            children: [
+              Checkbox(value: taskStatus, onChanged: onChanged),
+              const SizedBox(width: 10),
+              Text(
+                taskname,
+                style: TextStyle(
+                  decoration: taskStatus ? TextDecoration.lineThrough : TextDecoration.none,
```

Comment:
This code is too long (dart maximum is 80 characters wide), your formatter should have already split the lines